Source/WebKit/UIProcess/Inspector/win/RemoteWebInspectorUIProxyWin.cpp

```diff
@@ -131,9 +134,48 @@ WebPageProxy* RemoteWebInspectorUIProxy::platformCreateFrontendPageAndWindow()
     return m_webView->page();
 }
 
+void RemoteWebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&& saveDatas, bool forceSaveAs)
+{
+    ASSERT(saveDatas.size() == 1);
+    UNUSED_PARAM(forceSaveAs);
+
+    // Remove custom URI schemes such as "web-inspector://" and also remove the leading "/".
+    URL url { saveDatas[0].url };
+    Vector<WCHAR, MAX_PATH> filePath(url.path().substring(1).toString().wideCharacters());
+
+    OPENFILENAME ofn;
+    memset(&ofn, 0, sizeof(ofn));
+
+    ofn.lStructSize = sizeof(OPENFILENAME);
+    ofn.hwndOwner = m_frontendHandle;
+    ofn.lpstrFile = filePath.data();
+    ofn.nMaxFile = MAX_PATH;
+    ofn.Flags = OFN_PATHMUSTEXIST | OFN_OVERWRITEPROMPT;
+
+    if (GetSaveFileName(&ofn)) {
+        auto fd = openFile(filePath.data(), FileOpenMode::ReadWrite);
+        if (!isHandleValid(fd))
+            return;
+
+        String content { saveDatas[0].content };
+        auto contentSize = content.utf8().length();
+        auto bytesWritten = writeToFile(fd, content.utf8().data(), contentSize);
```

Comment:
`content.utf8()` allocates a buffer. Don't do it twice.
> auto content = saveDatas[0].content.utf8().

Reply:
Fixed at 19fcde7